module/zfs/zfs_log.c

```diff
@@ -936,5 +936,5 @@ zfs_log_clone_range(zilog_t *zilog, dmu_tx_t *tx, int txtype, znode_t *zp,
 	}
 }
 
-ZFS_MODULE_PARAM(zfs, zfs_, immediate_write_sz, S64, ZMOD_RW,
+ZFS_MODULE_PARAM(zfs, zfs_, immediate_write_sz, UINT, ZMOD_RW,
```

Comment:
There is also `zvol_immediate_write_sz` for zvols, which is `ssize_t` and has no module parameter.  Would be nice to sync them.

Reply:
Yeah, for sure. I have more changes coming to unify tunables across platforms, so I'll have a look at it as part of that, or after. Though I see zvol tunables are getting love elsewhere, so maybe I'll be beaten to it! :)